organizations/templates/organizations/shift_confirm_delete.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'base.html' %}
+{% load i18n %}
+
+{% block title %}{{ user.username }}{% endblock %}
+{% block body_class %}helpdesk{% endblock %}
+
+{% block content %}
+
+<form action="" method="post">{% csrf_token %}
+    <p>Are you sure you want to delete shift "{{ object }}"?</p>
+    <input type="submit" value="Confirm" />
+    <a href="{% url "shift_management" %}">Cancel</a>
+</form>
```

Comment:
I decided to use a confirmation page instead of a bootstrap dialogue because it was easier and faster to implement.